gradle.properties

```diff
@@ -1 +1 @@
-version=4.0.0
+version=5.0.0
```

Comment:
As we're modifying the constructors of two classes, this counts as a backwards incompatible change to me, so requires a major version bump.